lib/sedna/backend/tensorflow/__init__.py

```diff
@@ -74,7 +75,7 @@ def predict(self, data, **kwargs):
         if not self.has_load:
             tf.reset_default_graph()
             self.sess = self.load()
-        return self.estimator.predict(data=data, **kwargs)
+        return self.estimator.predict(data, **kwargs)
```

Comment:
If this is a modification of a parameter, it is an interface change and requires caution;
If this is not a modification of a parameter, there is no necessary.

Reply:
To be compatible with the naming used by developers for prediction,change this key parameter to arg